requirements.txt

```diff
@@ -0,0 +1,6 @@
+requests==2.20.0
```

Comment:
Probably resync this is already in the repo

Reply:
Do you mean that this dependency is transitive and is brought by another one? We don't have `resync` package in here, right?